pl_examples/full_examples/semantic_segmentation/semseg.py

```diff
@@ -0,0 +1,149 @@
+import os
+from argparse import ArgumentParser
+from collections import OrderedDict
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision
+import torchvision.transforms as transforms
+from torch.utils.data import DataLoader, Dataset
+from torchvision.models.segmentation import fcn_resnet50
+
+from PIL import Image
```

Comment:
pls move PIL to the block above with "3rd-party packages"

Reply:
Done in the latest commit.